tests/unit/fem/test_var_order.cpp

```diff
@@ -1081,7 +1088,14 @@ static void TestSolveParVec(ParFiniteElementSpace &fespace)
 
    ParGridFunction x(&fespace);
    x = 0.0;
-   x.ProjectBdrCoefficient(exsol, ess_attr);
+   if (x.BdrVectorDim() == 1)
+   {
+      x.ProjectBdrCoefficientNormal(exsol, ess_attr);
+   }
+   else
+   {
+      x.ProjectBdrCoefficientTangent(exsol, ess_attr);
+   }
```

Comment:
If, as I suggested above, we update `ProjectBdrCoefficient` to handle H(div) and H(curl) spaces, this change will not be necessary.